TESTING.md

```diff
@@ -0,0 +1,136 @@
+# Testing
+
+![C++](https://img.shields.io/badge/c++-%2300599C.svg?style=for-the-badge&logo=c%2B%2B&logoColor=white)
+![Python](https://img.shields.io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54)
+
+
+We are proud to present the user/interoperability testing framework we used for our CLI chat app, based on [OLAF-Neighbourhood protocol](https://github.com/xvk-64/2024-secure-programming-protocol).
+
+This document contains our testing suite and approaches for user/iteroperability testing.
+
+## Test List
```

Comment:
Should this maybe be a "Requirements List"?

Reply:
I think "Test List" captures the idea better, this is a testing document after all, and we are testing for them.